src/docs/reference/README.md

```diff
@@ -55,6 +55,7 @@ Here's a categorized overview of all built-in Orchard Core features at a glance.
   - [Flow](modules/Flow/README.md) 
   - [Bag](modules/Flow/BagPart.md)
   - [Publish Later](modules/PublishLater/README.md)
+  - [SEO Meta](modules/Seo/README.md)
```

Comment:
Seems there's an extra space before the hypen, same for above modules

Reply:
These are intentionally an indented section of the list.